### ISSUE TITLE
Clean up the repository

### ISSUE THREAD

**Issue by mrvivacious** [mrvivacious/PorNo-_Porn_Blocker#58]
**As a** software developer,
**I want** the codebase I work on to be clean and easy to navigate
**So that** there is one less obstacle in the way of pushing updates and features

**Work description**:
(This is only for the repository as a whole. The codebase is an entirely different issue, both figuratively and literally)

- [ ] Finish writing the readme
- [ ] Delete [porNo_opensource](https://github.com/mrvivacious/PorNo-_Porn_Blocker/blob/master/porNo_opensource.js) file (make a note in readme to look up JS implementation in chromeExtension/)
- [ ] OR, instead of deleting, make an `archive/` folder and put deprecated assets there as memorabilia
- [ ] Put pictures in `assets/` ?

In summary, the root directory should have directories and "essential" files only (readme, license etcetera).

**Comment by mrvivacious** [comment]
Made an `archive/` folder. Moved porNo_opensource file to that. Put pictures in `pictures/` for the time being

**Comment by mrvivacious** [comment]
for the Readme,

Contact me:
add button to email me jvnnvt@gmail.com. Use mailto (html)
Link to my instagram and welcome DM

Edit the URL lists:
explain where the urls live. "open in text editor. add or delete site in the following format. run ./generateLists.py."

Replace `Chrome Extension
Click this or click the "Available in the Chrome Web Store" button at the top of the README` with `Download live Chrome Extension` or smth. easier to read text
